### DISCUSSION TITLE
Any chance a new release could be cut?

### DISCUSSION THREAD

**Discussion by dbt** [Microsoft/node-pty#743] [Q&A]
It's been a while since the last one, and latest release still has an issue when running against python 3.12 due to distutils that was fixed last march in #646

**Comment by Tyriar** [comment]
I'd recommend using the `beta` tag if you need recent fixes. Releases are fairly infrequent for this as it's a bit of a pain to do. We'll probably do one once https://github.com/microsoft/node-pty/issues/490 lands.

**Reply by mattieb** [reply]
Hey, any chance we look at #627 too?

I'm still using my own fork just for the argv[0] support and I've had to catch up to conflicting changes a couple times now, most recently a few days ago.
